### DISCUSSION TITLE
Module Import Issue - Cannot Be Loaded from Menu Bar

### DISCUSSION THREAD

**Discussion by kentaro-963** [cwapi3d/cwapi3dpython#193] [Q&A] [answered]
Hi,

I am encountering an issue with module imports in my Python script. 
When I run the script from the IDLE Shell, it works perfectly. 
However, when I try to run the same script from the menu bar, the module imports fail.

Is module import supported when running scripts from the menu bar? 
If so, could you provide guidance on how to resolve this issue?

directory 
```
userprofil_30/
└── 3d/
    └── API.x64/
        └── test/
            ├── test.py
            └── module.py
```

test.py
```
import utility_controller as uc 
from module import create_node

user_coord_x = uc.get_user_int("coord_x")
user_coord_y = uc.get_user_int("coord_y")
user_coord_z = uc.get_user_int("coord_z")

node = create_node(user_coord_x, user_coord_y, user_coord_z)
```

module.py
```
import cadwork as cw
import element_controller as ec

def create_node(coord_x, coord_y, coord_z):
    point = cw.point_3d(coord_x, coord_y, coord_z)
    return ec.create_node(point)
```

**Comment by Brunner246** [comment]
Hi @kentaro-963 
See https://github.com/cwapi3d/cwapi3dpython/discussions/118#discussioncomment-8232882 , this should show how to load other modules.

**Reply by kentaro-963** [reply, marked as answer]
Hi @Brunner246 
Thank you for your prompt response and assistance.

I am pleased to report that the issue with module imports has been resolved. The guidance you provided was very helpful.

Thank you once again for your support.
